front-end/src/App.js

```diff
@@ -26,12 +27,12 @@ function App() {
       <Router>
         <Header />
         <Switch>
-          <Route exact path="/" component={Gallery}/>
+          <Route exact path="/" component={Gallery} />
           {/* <Route exact path="/signIn" component={LoginPage} /> */}
           <Route exact path="/signIn" render={(props) => user?.username ? <ErrorLog /> : <LoginPage {...props} />} />
-
-          <Route exact path="/add" component={AddProduct}/>
```

Comment:
Make add, edit and delete routes only for signed users. Use HOK to wrap the Route component.